### DISCUSSION TITLE
Proposal to Enhance Keyboard Shortcut Capabilities: Adding Command Key Support

### DISCUSSION THREAD

**Discussion by truongvan** [helix-editor/helix#7063] [General]
Dear Helix Editor Community,

I propose the integration of Command key support into Helix Editor's existing shortcut system, expanding usability for MacOS users and widening potential shortcut combinations.

Currently, Helix Editor supports Ctrl, Option, and Shift modifiers for shortcuts. The addition of the Command key would:

1. Enhance consistency with typical MacOS keyboard interactions.
2. Expand shortcut possibilities by doubling potential combinations.
3. Improve customizability for users tailoring their shortcut mappings.

This feature would not replace or disrupt the current system but would offer an optional enhancement that avoids conflicts with existing Command key functionality on MacOS. I believe this feature would significantly benefit the Helix Editor user base.

Best,

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/issues/6582
